templates/blog/articleCreate.html.twig

```diff
@@ -0,0 +1,36 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Création d'article
+{% endblock %}
+
+{% block body %}
+	{{ form_start(form) }}
+
+	<fieldset>
+		<legend>Créer un article</legend>
+
+		{{ form_row(form.title, {
+         'label': 'Titre de l\'article',
+         'attr': {
+           'placeholder':'Titre'
+    }
+    }) }}
+
+			{{ form_row(form.capon, {
+        'label': 'Chapon',
+		'attr': {
+          'placeholder':'Sous-titre de l\'article'
+    }}) }}
+
+		{{ form_row(form.content, {
+        'label': 'Contenu de l\'article',
+        'attr':{
+          'placeholder':'Contenu de l\'article'
+    }}) }}
+
+    <button class="btn btn-primary" type="submit">
+        Crée un article
+    </button>
+
+		{{ form_end(form) }}
+	{% endblock %}
```

Comment:
You should check code style in this file , it's very hard to read.

To help you , you could use this following lib : https://github.com/friendsoftwig/twigcs